ibis/backends/tests/test_export.py

```diff
@@ -43,7 +43,6 @@ def no_pyarrow(backend):
                     "impala",
                     "pandas",
                     "pyspark",
-                    "snowflake",
```

Comment:
obviously having CI pass is the more important issue here, but if we skip the snowflake tests, does it cause failures if it's marked in `notimpl`?  

My minor nit with removing it from here is that it looks like this _is_ implemented for `snowflake`

Reply:
> obviously having CI pass is the more important issue here, but if we skip the snowflake tests, does it cause failures if it's marked in notimpl?

In this case the tests are passing. The test will only run if `SNOWFLAKE_URL` is defined, otherwise it's skipped.

The existing semantics around `notimpl` are no different when `SNOWFLAKE_URL` is defined.

Reply:
The way I think about it after this PR is "whether something is notimpl/notyet/broken is unknowable if the test is skipped".

An alternative would be to require use of `not snowflake` if you don't have the environment variable defined.

Another alternative might be to skip discard snowflake backend tests from discovery if the environment variable isn't defined.

Reply:
@gforsyth Anything else you want me to do hereE?

Reply:
no, I think we're good -- thanks for the clarification